site/_posts/2017-04-10-Announcing-Euwe-2.1.html.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Announcing Euwe-2.1
+author: jprause
+date: 2017-04-10 08:35:39 UTC
+comments: true
+published: true
+tags: releases announcements
+---
+
+We've just built Euwe-2.1. This release resolves the issue in Euwe-2, where dropdowns and trees don't work in the UI. There is also a manual fix found [here](http://talk.manageiq.org/t/euwe-2-broken-ui-fix/2221).
+
+| Appliance | Format | Size |
+| --------- | ------ | ---- |
+| [Amazon EC2](http://releases.manageiq.org/manageiq-ec2-euwe-2.1.vhd) | ec2 | 3.3 GB |
+| [Docker](https://hub.docker.com/r/manageiq/manageiq/) | docker | 0.5 GB |
+| [Google Compute Engine](http://releases.manageiq.org/manageiq-gce-euwe-2.1.tar.gz) | gce | 1.0 GB |
+| [libvirt](http://releases.manageiq.org/manageiq-libvirt-euwe-2.1.qc2) | libvirt | 1.1 GB |
+| [Microsoft Azure](http://releases.manageiq.org/manageiq-azure-euwe-2.1.vhd) | azure | 3.3 GB |
+| [Microsoft SCVMM](http://releases.manageiq.org/manageiq-hyperv-euwe-2.1.vhd) | hyperv | 3.3 GB |
+| [Openstack](http://releases.manageiq.org/manageiq-openstack-euwe-2.1.qc2) | openstack | 1.1 GB |
+| oVirt [ova](http://releases.manageiq.org/manageiq-ovirt-euwe-2.1.ova) or [qc2](http://releases.manageiq.org/manageiq-ovirt-euwe-2.1.qc2) | ovirt | 1.1 GB |
+| [Qemu/KVM](http://releases.manageiq.org/manageiq-openstack-euwe-2.1.qc2) | openstack | 1.1 GB |
+| Red Hat Enterprise Virtualization [ova](http://releases.manageiq.org/manageiq-ovirt-euwe-2.1.ova) or [qc2](http://releases.manageiq.org/manageiq-ovirt-euwe-2.1.qc2) | ovirt | 1.1 GB |
```

Comment:
We need to change the download page too, but the product is now called "Red Hat Virtualization".

`Red Hat Virtualization (ova or qc2)` ?